wemake_python_styleguide/options/defaults.py

```diff
@@ -57,6 +57,9 @@
 #: Maximum number of imports in a single module.
 MAX_IMPORTS: Final = 12
 
+#: Maximum number of imported names in a single module.
+MAX_IMPORTED_NAMES: Final = 50
```

Comment:
Can you please write a comment about how this value was created? Like here https://github.com/wemake-services/wemake-python-styleguide/blob/dc96ff3dea3c5b759cd93bbb681702673b7b607f/wemake_python_styleguide/options/defaults.py#L55